src/main/java/com/assylias/jbloomberg/IntradayBarRequestBuilder.java

```diff
@@ -94,8 +94,8 @@ public IntradayBarRequestBuilder fillInitialBar() {
     /**
      * Adjust historical pricing based on the DPDF&lt;GO&gt; BLOOMBERG PROFESSIONAL service function.
      */
-    public IntradayBarRequestBuilder adjustDefault() {
-        this.adjustDefault = true;
+    public IntradayBarRequestBuilder noAdjustDefault() {

```

Comment:
Well spotted!

Re-reading the Bloomberg docs, if this is set to true, all other settings are ignored. So maybe a better way would be to set the field to false on line 33 and leave the method as is. I'll improve the javadoc to clarify the behaviour.

Reply:
From what I read from the docs the bloomberg defaults to true for this setting.
If initially set to false then the default jbloomberg request would result in a non default bloomberg one.
Are you sure?

Reply:
That's a valid point. The problem with your change is that `new IntradayBarRequestBuilder().adjustSplits()` will ignore the `adjustSplits` call and rely on the default config (which may or may not adjust splits).

One solution would be to keep your change and turn `adjustDefault` to false in `adjustSplits`, `adjustNormalDistributions` and `adjustAbnormalDistributions` and clarify in the javadoc of those methods that they turn the defaults off.

What do you think?

Reply:
`ignorePricingDefaults()` would probably be a better name.

Reply:
And I realise that `HistoricalRequestBuidler` needs to be fixed too. If it's taking too much of your time I can accept the PR as is and make changes at my end as a new commit.

Reply:
How about making 
`adjustSplits()`, `adjustNormalDistributions()` and `adjustAbnormalDistributions()` also set `adjustDefault = false`?
and also mention it in the docs?

Reply:
Yes agreed.

Reply:
In that case, does it make sense to have an `ignorePricingDefaults()` method?
They will be automatically ignored as soon as the user requests something different.

Reply:
It's useful if you want no price or volume adjustment at all.

Reply:
If I do nothing, with your code, Bloomberg defaults will be used. It's fine, I'll just merge your code as is and make some additional changes. Thanks again for your help.

Reply:
Thanks a lot for making jbloomberg! It really helped me a lot.
Before you release, please consider also fixing the bbcomm.exe location.